src/extras/MovieClip.js

```diff
@@ -56,13 +56,13 @@ function MovieClip(textures)
     this.onComplete = null;
 
     /**
-     * The MovieClips current frame index (this may not have to be a whole number)
+     * Elapsed time since animation has been started, used internally to display current texture
```

Comment:
i was wondering about that, but you can still set/get this value that's why i used name without underscore.